src/schema/updates/action.json

```
@@ -1,3 +1,9 @@
+// The "action" API namespace is currently part of browser_action.json
+// along with "browserAction", the following is needed to make sure we
+// will normalize and import the "action" API namespace in a file named
+// "src/schema/imported/action.json" (similar to what contextMenus.json
+// does for the "contextMenus" API namespace which is defined along with
+// the "menus" API namespaces).<<
```

```suggestion
// the "menus" API namespaces).
```

Does `<<` serve any purpose?